webapp-django/crashstats/api/tests/test_views.py

```diff
@@ -171,6 +173,28 @@ def mocked_get(url, params, **options):
         eq_(response.status_code, 200)
         eq_(response['Access-Control-Allow-Origin'], '*')
 
+    def test_no_cookie_set(self):
+
+        def mocked_get(**options):
+            return {}
+
+        # Doesn't matter which model we use, as long as we use one of them.
+        CrontabberState.implementation().get.side_effect = mocked_get
+        url = reverse('api:model_wrapper', args=('CrontabberState',))
+        # self.client.handler.enforce_csrf_checks = True
+        response = self.client.get(url)
+        eq_(response.status_code, 200)
+        # The test client sucks up all set cookies. This one should not
+        # be set.
+        ok_(ANON_COOKIE not in response.cookies)
+
+        # But load the regular web page.
+        # The documentation for the API
+        url = reverse('api:documentation')
```

Comment:
This PR is about nixing the anon csrf stuff for API views, but isn't this an API view? It reads confusingly.